<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADV_serial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>COM port</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Result</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
       <property name="flags">
        <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
       <item>
        <property name="text">
         <string>Round</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Write byte</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Read byte</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>COM2</string>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
       <item>
        <property name="text">
         <string>Round</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Write byte</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Read byte</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>286</width>
     <height>361</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Serial Setting</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>COM1</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>84</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Baud Rate</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox1">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>108</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>American Typewriter</family>
           <pointsize>13</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="currentText">
          <string>9600</string>
         </property>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>75</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>110</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>150</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>230400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>460800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>921600</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>COM Port name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="comName1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="startButton1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="closeButton1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>420</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>COM2</string>
        </attribute>
        <widget class="QPushButton" name="startButton2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="closeButton2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>420</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>84</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Baud Rate</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>108</width>
           <height>27</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>American Typewriter</family>
           <pointsize>13</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="currentText">
          <string>9600</string>
         </property>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>75</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>110</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>150</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>230400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>460800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>921600</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Futura</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>COM Port name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="comName2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <zorder>startButton2</zorder>
        <zorder>closeButton2</zorder>
        <zorder>treeWidget</zorder>
        <zorder>label_3</zorder>
        <zorder>comboBox2</zorder>
        <zorder>label_4</zorder>
        <zorder>comName2</zorder>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
